src/scrabble.js

```diff
@@ -1,5 +1,42 @@
-function scrabble() {
-
+const alphabetData = {
+  a: 1,
+  e: 1,
+  i: 1,
+  o: 1,
+  u: 1,
+  l: 1,
+  n: 1,
+  r: 1,
+  s: 1,
+  t: 1,
+  d: 2,
+  g: 2,
+  b: 3,
+  c: 3,
+  m: 3,
+  p: 3,
+  f: 4,
+  h: 4,
+  v: 4,
+  w: 4,
+  y: 4,
+  k: 5,
+  j: 8,
+  x: 8,
+  q: 10,
+  z: 10
 }
 
+function scrabble(word) {
+  let score = 0
+  if (word === null) {
+    return 0
+  }
+  const alphaNum = word.toLowerCase().split('')
+  alphaNum.forEach((alphaNum) => {
```

Comment:
You could maybe change the variable name inside forEach for easier readability- letter for example. Otherwise you have alphaNum twice referring to both the whole array + individual array items. But otherwise really neat and tidy implementation